app/apps/server/bridges/cloud.ts

```diff
@@ -0,0 +1,20 @@
+import { Meteor } from 'meteor/meteor';
+import { ICloudWorkspaceBridge } from '@rocket.chat/apps-engine/server/bridges';
+
+import { getWorkspaceAccessToken } from '../../../cloud/server';
+import { AppServerOrchestrator } from '../orchestrator';
+
+const boundGetWorkspaceAccessToken = Meteor.bindEnvironment((scope: string) => getWorkspaceAccessToken(false, scope, false));
```

Comment:
@geekgonecrazy this call returns a token, but I'm not sure how to check its validity 😛

Reply:
This needs true I think minimum as first arg.  Otherwise it'll just return what ever token it has saved

Reply:
But honestly I think we need another function here that maybe this functions wraps too.. that returns expire and token and gives option to save the token.  This way the apps can know when a token will expire.  Also since it will request a single scope we won't want it to be saved